primitiv/_device.pyx

```diff
@@ -24,7 +26,18 @@ cdef class Device:
         :type device: primitiv.Device
 
         """
-        CppDevice.set_default(device.wrapped[0])
+        global py_primitiv_default_device
+        py_primitiv_default_device = device
```

Comment:
Type check?

Reply:
Not required. The argument is typed statically, and it throws `TypeError` if a wrong argument is specified.
```
In [5]: Device.set_default({})
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-558a1bb96fcd> in <module>()
----> 1 Device.set_default({})

TypeError: Argument 'device' has incorrect type (expected primitiv._device.Device, got dict)
```